inc/settings.php

```diff
@@ -284,6 +284,12 @@ function settings_fields( $fields ) {
 			'description' => __( 'How many pixels of scrolling result in a single pixel of parallax motion. 0 means automatic. Lower values give more noticeable effect.', 'siteorigin-panels' ),
 		);
 
+		$fields['general']['fields']['parallax-mobile'] = array(
+			'type'        => 'checkbox',
+			'label'       => __( 'Disable Background Parallax On Mobile', 'siteorigin-panels' ),
```

Comment:
The setting above is "Limit Parallax Motion". What do you think about changing this label to "Disable Parallax On Mobile" ?